akka-javasdk-testkit/src/main/java/akka/javasdk/testkit/EventSourcedTestKit.java

```diff
@@ -41,7 +52,29 @@ private EventSourcedTestKit(ES entity, String entityId) {
    */
   public static <S, E, ES extends EventSourcedEntity<S, E>> EventSourcedTestKit<S, E, ES> of(
       Supplier<ES> entityFactory) {
-    return of("testkit-entity-id", entityFactory);
+    return of(DEFAULT_TEST_ENTITY_ID, entityFactory);
+  }
+
+  /**
+   * Creates a new testkit instance from a Supplier of EventSourcedEntity and a state into which
+   * the supplied entity will be placed for tests.
+   *
+   * <p>A default test entity id will be automatically provided.
+   */
+  public static <S, E, ES extends EventSourcedEntity<S, E>> EventSourcedTestKit<S, E, ES> ofEntityWithState(
+      Supplier<ES> entityFactory, S initialState) {
+    return ofEntityWithState(DEFAULT_TEST_ENTITY_ID, entityFactory, initialState);
+  }
+
+  /**
+   * Creates a new testkit instance from a Supplier of EventSourcedEntity and events from which to
+   * derive a state for the generated entity for tests.
```

Comment:
```suggestion
   * Creates a new testkit instance from a Supplier of EventSourcedEntity and events from which to
   * derive a state for the generated entity.
```

We can omit 'for tests' since we have already enough context.